application-analytics-ui/src/main/resources/Analytics/Code/AnalyticsTranslations.xml

```diff
@@ -46,16 +46,36 @@ Analytics.Code.AnalyticsDashboardClass_endDate.hint=End time of the date interva
 Analytics.Code.AnalyticsDashboardClass_description.hint=Additional information to be displayed.
 Analytics.Code.AnalyticsDashboardClass_DisplayedStatistics.hint=List of statistics widgets that will be displayed in the current dashboard, in order.
 
+#Config
+Analytics.Code.ConfigurationClass.matomoTrackingCode.hint=Tracking code received from Matomo. The 'u' variable might be different than the one that is initially generated by Matomo if the server you want to track is on another machine. To fix this, go to the integration tab, copy the Matomo URL, and replace the value of 'u' with the Matomo URL.
+Analytics.Code.ConfigurationClass.idSite.hint=The ID of the site that you want to see the statistics for (found in the integration tab).
+Analytics.Code.ConfigurationClass.requestAddress.hint=The Matomo url (found in the integration tab).
+Analytics.Code.ConfigurationClass.authToken.hint=Token generated by Matomo to be capable of using the API to view the data.
+
+#Most Viewed Pages 
+mostViewedPages.label=Page Title
+mostViewedPages.nb_visits=Unique Vistators
+mostViewedPages.nb_hits=Page Views
+mostViewedPages.sum_time_spent=Time spent(s)
+mostViewedPages.entry_nb_visits=No. of views that started on this page
+mostViewedPages.entry_nb_actions= No. Of Actions
+mostViewedPages.bounce_rate=Bounce Rate
+mostViewedPages.exit_rate=Exit Rate
```

Comment:
Are these titles taken from the Matomo UI? If not, can we do that?
It would be helpful to also a screenshot of the Matomo UI for this widget

Reply:
The translated names are from the Matomo API documentation, not the UI. I attached a screenshot of the Matomo UI names.
![image](https://github.com/xwikisas/application-analytics/assets/92780090/a3d4dcf4-e86e-4489-8f63-815e68c35e3f)